.github/workflows/test.yml

```diff
@@ -0,0 +1,41 @@
+name: build
+
+on: [ push, pull_request ]
+
+jobs:
+  run:
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        operating-system: [ ubuntu-latest ]
+        php-versions: [ '7.3', '7.4', '8.0' ]
+    name: PHP ${{ matrix.php-versions }} Test on ${{ matrix.operating-system }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-versions }}
+          extensions: mbstring, zip
```

Comment:
Just a thought... Wouldn't it be useful to mention the following in there? 

```
   tools: phpunit, composer:v2
```

Reply:
https://github.com/marketplace/actions/setup-php-action#wrench-tools-support

Reply:
I think this is not useful because we cannot specify the PHPUnit version. That's the reason why I add required extensions for `Composer` to run `composer update` command conveiently.

Reply:
I'm good with that 👍